lib/Dialect/TritonNvidiaGPU/Transforms/RewriteTensorPointer.cpp

```diff
@@ -741,7 +772,8 @@ class TritonGPURewriteTensorPointerPass
   }
 
   void visitOperation(Operation *op, std::stack<Operation *> &eraser,
-                      DenseSet<Value> &valueToRemove) {
+                      DenseSet<Value> &valueToRemove, ModuleOp module) {
+    assert(!!module);
```

Comment:
why `!!`?

Reply:
lol, in email it looked like `why !!?`, which was also a fair reaction.  I should get rid of `module` entirely.  Done, thanks.